### PR TITLE
Add Initial PGO File

### PR DESCRIPTION
**Pull request by nibanks**
This adds the PGD file for msquic.dll that resulted after a single round (client & server) of PGO on a single connection throughput test. It seems to have improved max throughput a little bit.

### CODE REVIEW

**Comment by anrossi** [comment]
Is the .pgd file actually binary data, or do we need to add another rule to `.gitattributes` to appropriately store the file?

File: CMakeLists.txt [hunk 1]
```diff
@@ -112,7 +112,7 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 
     set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} /GL /Zi")
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /GL /Zi")
-    set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /LTCG /DEBUG /OPT:REF /OPT:ICF")
+    set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /LTCG /USERPROFILE:PGD=${CMAKE_CURRENT_SOURCE_DIR}/src/bin/winuser/msquic.pgd /DEBUG /OPT:REF /OPT:ICF")
```
Comment (anrossi) [inline CMakeLists.txt:115]: does `/DEBUG` belong in the release flags?

**Comment by nibanks** [comment]
> Is the .pgd file actually binary data, or do we need to add another rule to `.gitattributes` to appropriately store the file?

It is binary.

File: CMakeLists.txt (hunk 1 above)
Comment (nibanks) [reply CMakeLists.txt:115]: The `/DEBUG` flag just causes `.pdb` files to be generated.

**Pull request merged**